utils/ansible.py

```diff
@@ -0,0 +1,281 @@
+from distutils.sysconfig import get_python_lib
+from os import listdir, mkdir, path, remove
+from shutil import copy, copyfile, rmtree
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from fauxfactory import gen_alphanumeric
+from utils import conf
+from utils.providers import get_crud
+
+from git import Repo
+from yaml import load, dump
+
+
+yml_path = path.join(path.dirname(__file__) + "/manageiq_ansible_module/")
+library_path = get_python_lib()
+basic_yml_path = path.dirname(__file__) + "/ansible_conf/"
```

Comment:
I tried this but it's challenging to work with a tmpdir that is managed by pytest because I have to create a new one in every test function. Can I create a regular python temp directory? 
dirpath = tempfile.mkdtemp()
?